ipmininet/router/__router.py

```diff
@@ -126,7 +127,11 @@ def start(self):
             self._old_sysctl[opt] = self._set_sysctl(opt, val)
         # Fire up all daemons
         for d in self.nconfig.daemons:
-            self._processes.popen(shlex.split(d.startup_line))
+            if (hasattr(d,'logfile')):
```

Comment:
This feature should always be there, and not only for the daemons have a logfile attribute.

I'd suggest to always log stderr/stdout of all daemons, e.g.,
- Create a temp logdir using daemon_logs tempfile.TemporaryDirectory(dir='tmp')
- display it on the global logger
- capture stdout/stderr for all daemon, e.g. stderr='%s/%s_%s_err.log' % (daemon_log, node, daemon_name)